src/selectors/getForm.js

```diff
@@ -0,0 +1,44 @@
+import {
+  getFormAsyncErrors,
+  getFormError,
+  getFormInitialValues,
+  getFormMeta,
+  getFormSubmitErrors,
+  getFormSyncErrors,
+  getFormSyncWarnings,
+  getFormValues,
+  hasSubmitFailed,
+  hasSubmitSucceeded,
+  isDirty,
+  isInvalid,
+  isPristine,
+  isSubmitting,
+  isValid,
+} from 'redux-form';
+
+/**
+ * Returns a complete representation of a redux form by
+ * aggregating results of all other selectors into one object.
```

Comment:
@gdbrown there is actually one selector (getFormNames) that is not included here, because it is not form-specific. Probably not important enough to note that here, but fyi.

see https://redux-form.com/7.2.1/docs/api/selectors.md/#-code-getformnames-code-returns-code-state-gt-formnames-array-code-